.storybook/preview.tsx

```diff
@@ -0,0 +1,14 @@
+// .storybook/preview.tsx
+
+import React from 'react';
+import { QueryClientProvider } from 'react-query';
+import { queryClient } from './queryClient';
+import { Decorator } from '@storybook/react';
+
+export const decorators: Decorator[] = [
+  (Story) => (
+    <QueryClientProvider client={queryClient}>
```

Comment:
Z tego nie będziemy korzystać to można to wyrzucić na ten moment QueryClientProvider.